src/languages/en.js

```diff
@@ -476,8 +476,8 @@ export default {
     },
     cameraPermissionsNotGranted: 'Camera permissions not granted',
     messages: {
-        noPhoneNumber: 'Please enter a phone number including the country code e.g +447814266907.',
-        errorMessageInvalidPhone: 'That doesn’t look like a valid phone number. Try again with the country code. e.g. +15005550006',
+        noPhoneNumber: 'Please enter a valid phone number without brackets or dashes. If you\'re outside the US please include your country code, eg. +447782339811',
```

Comment:
Let's not have duplicated text. Let's just keep `errorMessageInvalidPhone` and remove `noPhoneNumber`. Same time let's update the code usages of `noPhoneNumber` to `errorMessageInvalidPhone`.

Reply:
Done